examples_tests/52.SystemTest/main.cpp

```diff
@@ -302,6 +302,14 @@ int main(int argc, char** argv)
 		}
 	};
 	
+	auto arch = system->createFileArchive("test.zip");
```

Comment:
why is createArchive rolled in together with opening the mapped file it will read from?